<?xml version="1.0" encoding="utf-8"?>
<launch>

  <arg name="computer_name" value="usl" />
  <param name ="/use_sim_time" value="true"/>
  <node  pkg="projection" name="left_projection_node" type="projection_node" output="screen">
    <remap from="/image_in" to="/nerian_stereo/left_image"/>
    <remap from="/cloud_in" to="/velodyne_points"/>
    <param name ="result_str" value="/home/$(arg computer_name)/catkin_ws/src/camera_lidar_calibration/calibration/debug_data/C_T_L_left.txt"/>
    <param name ="cam_config_file_path" value="/home/$(arg computer_name)/catkin_ws/src/camera_lidar_calibration/config/nerian_passthru_faccal_left.yaml"/>
  </node>

</launch>
